compiler-rt/lib/tsan/rtl/tsan_platform.h

```diff
@@ -697,6 +726,8 @@ ALWAYS_INLINE auto SelectMapping(Arg arg) {
   return Func::template Apply<MappingGoS390x>(arg);
 #  elif defined(__aarch64__)
   return Func::template Apply<MappingGoAarch64>(arg);
+#  elif defined(__loongarch64)
```

Comment:
` #  elif defined(__loongarch64)`
Please use `__loongarch_lp64` instead of `__loongarch64`.

Reply:
Done, Thanks.